FABulous/FABulous_API.py

```diff
@@ -203,7 +202,7 @@ def genBitStreamSpec(self):
         specObject = self.fabricGenerator.generateBitsStreamSpec()
         return specObject
 
-    def genModelNpnr(self):
+    def genModel(self):
```

Comment:
I still think it's worth having a function name that clarifies this is a model for nextpnr, just `genModel` is pretty vague